DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/PerturbProcessor.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
     public class PerturbProcessor : IAnonymizerProcessor
     {
         private PerturbFunction _perturbFunction;
-        private static readonly HashSet<string> _supportedVR = Enum.GetNames(typeof(PerturbSupportedVR)).ToHashSet(StringComparer.InvariantCultureIgnoreCase);
+        private static readonly HashSet<DicomVR> _supportedVR = DicomDataModel.PerturbSupportedVR;
```

Comment:
**[Suggestion]**
Since `DicomDataModel` is a public data structure, seems the `_supportedVR` is not necessary here.
You can directly use `DicomDataModel.PerturbSupportedVR` in `IsSupported` function.

Same suggestion to XXXXProcessor classes.